Felienne/hedy
# [BUG FIX] Highscores country filtering no longer crashes

**Description**
In this PR we fix the highscores counter filtering. For unknown reasons there seems to be usernames that return a none value when requesting the user data. This shouldn't be possible, but it makes the page crash. We surround this by a try-except to catch the error. We also improve the code a bit and change a table header to make more sense.

**Possible issue**
The only possible issue I can think of is that a user removed their profile but their public profile remained. This shouldn't be possible, but we have to dive into this later on.

**How to test**
Can't be tested, deploy to beta and verify that it works (Alpha the error didn't happen).
